content/resources/crownlabs_ssh/crownlabs_ssh.md

```diff
@@ -0,0 +1,97 @@
+
+3. A password will be asked. You are free to insert one or to ignore it.
+
+Now you will find in the previously chosen path (`$HOME/.ssh` if you chose the default path, `C:\Users\yourusername\.ssh`) a file with a **.pub** extension (if you chose the default file name you will find `id_rsa.pub`). This is the file containing your **SSH public key**.
+
+Print the content of your **SSH public key** running:
+
+`cat $HOME/.ssh/id_rsa.pub` or opening `C:\Users\yourusername\.ssh` with an editor in **windows**
+
+Now copy the **SSH public key**.
+
+## Upload your key on [CrownLabs](https://crownlabs.polito.it/)
+
+Now open your page on [**CrownLabs**](https://crownlabs.polito.it/) and go in the [**Account page**](https://preprod.crownlabs.polito.it/app/account).
+
+<br>
+
+1. In the central box you will see a tab called **SSH key**. Click it to navigate in the **SSH keys** **management** panel.
+
+![image-20211112184900823](./img/1_1.png)
+
+<br>
+
+2. Now click on **Add SSH key** to open a modal that will allow you to upload your key.
+
+![image-20211112185058163](./img/1_2.png)
+
+<br>
+
+3. Paste your **Public key** (previously copied) inside the modal and add a **Name** (whatever you want).
+
+![image-20211112190017165](./img/1_3.png)
+
+<br>
+
+4. Now click on **Save**, the modal will close and your **SSH public key** will be uploaded.
+
+![image-20211112190234270](./img/1_4.png)
+
+<br>
+
+## Connect to a VM from terminal
+
+<br>
+
+1. Create an instance of a VM in your [**CrownLabs Dashboard page**](https://preprod.crownlabs.polito.it/app).
+
+![image-20211112191418307](./img/2_1.png)
+
+<br>
+
+2. Now move to the **[Active page](https://preprod.crownlabs.polito.it/app/active)** and click on the **SSH** icon inside the row of the **VM** that you have just created. A popup with a shell command inside will be shown, **copy** that command.
+
+![image-20211112193803509](./img/2_2.png)
+
+<br>
+
+3. Now open a **terminal**, paste the previously copied command and run it.
+   A prompt asking if you trust the host you are trying to connect with will appear. Type **yes** and press **Enter**.
+   Then another prompt asking for a **password** will appear. Type **crownlabs** and press **Enter** to **login** inside your **VM**.
+
+![image-20211112194835635](./img/2_3.png)
+
+<br>
+
+## Conclusion
+
+Now you are able to log in your **VMs** using **SSH**. Note that the **generation** of the **SSH key pair** an the upload of the **SSH public key** doesn't have to be repeated each time you create a new **VM**, this process only needs to be done once.
```

Comment:
```suggestion
Now you are able to log in your **VMs** using **SSH**. Note that the **generation** of the **SSH key pair** and the upload of the **SSH public key** does not have to be repeated each time you create a new **VM**, this process only needs to be done once.
```

Reply:
Would also add that it is possible to add other keys following the same process to access the VMs from different PCs